README.md

```diff
@@ -1,3 +1,80 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone
+
+>In this capstone project I am going to build  based on an online website for a conference. With provided  design guidelines to create the website, but I will personalize the content, i.e., instead of a conference you can build a website for a concert or for a web development course.
+
+
+## Built With
+
+HTML
+CSS
+JAVASCRIPT
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+A computer and internet connect
+
+Clone Reprository
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] Kindly add instructions for how to clone your repo and run it in the readme.